homeassistant/components/plex/config_flow.py

```diff
@@ -62,6 +63,18 @@ def configured_servers(hass):
     }
 
 
+async def async_discover(hass):
```

Comment:
What is calling this?

Reply:
Nothing yet (besides the tests). The plan discussed is to offer integration-handled discovery as part of onboarding.

Reply:
Is there an architecture issue where this is described?

Reply:
Actually, this is already described pretty well in https://github.com/home-assistant/architecture/issues/172.

Reply:
I think those are just ideas. The api is not described from what I read.